src/handler/mod.rs

```diff
@@ -30,19 +30,21 @@ pub fn handle_rollback() -> Result<(), Error> {
     }
 }
 
-pub fn set_boot_counter(reboot_count: i32) -> Result<(), Error> {
+pub fn set_boot_counter(reboot_count: i32) -> Result<()> {
     match get_boot_counter() {
-        Some(_) => bail!("boot_counter already set"),
+        Some(_) => Ok(()),
```

Comment:
so this silences the log but is it ok?! should high level notice this and somehow do something? maybe we're here after a bad state and whatnot, do we know the extent of silencing this?!